### ISSUE TITLE
No other output for a long time.

### ISSUE THREAD

**Issue by Simons2017** [OpenNMT/OpenNMT-py#1616]
I am trying to train a summarization model (using the transformer), my script is:
`onmt_train -data data/cnndm/CNNDM \
           -save_model models/cnndm \
           -layers 4 \
           -rnn_size 512 \
           -word_vec_size 512 \
           -max_grad_norm 0 \
           -optim adam \
           -encoder_type transformer \
           -decoder_type transformer \
           -position_encoding \
           -dropout 0\.2 \
           -param_init 0 \
           -warmup_steps 8000 \
           -learning_rate 2 \
           -decay_method noam \
           -label_smoothing 0.1 \
           -adam_beta2 0.998 \
           -batch_size 4096 \
           -batch_type tokens \
           -normalization tokens \
           -max_generator_batches 2 \
           -train_steps 200000 \
           -accum_count 4 \
           -share_embeddings \
           -copy_attn \
           -param_init_glorot \
           -world_size 2 \
           -gpu_ranks 0 \
	   --report_every 1`
I got
`[2019-10-29 12:13:22,664 INFO] encoder: 35166720
[2019-10-29 12:13:22,664 INFO] decoder: 39417880
[2019-10-29 12:13:22,664 INFO] * number of parameters: 74584600
[2019-10-29 12:13:22,665 INFO] Starting training on GPU: [0]
[2019-10-29 12:13:22,665 INFO] Start training loop and validate every 10000 steps...
`
no other output for a long time. I check GPU and CPU and it shows %0-%6 GPU and 0%CPU. I am sure I install requirements properly, could you give me some advice?

**Comment by vince62s** [comment]
-world_size 2 \ -gpu_ranks 0
this is inconsistent. if you have 2 GPUs you need to set  0 1, if you have one, size is 1.

**Comment by Simons2017** [comment]
It works, thank you!
